catalog/admin/admin-poller-service/pom.xml

```diff
@@ -57,6 +57,11 @@
             <groupId>ddf.action.core</groupId>
             <artifactId>action-core-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.lib</groupId>
+            <artifactId>spock-shaded</artifactId>
+            <version>${ddf.version}</version>
```

Comment:
Ah, heck. You'd think I'd have noticed that. :(

Shaun is right. You'll need to add the full set of dependencies and the build plugin.